data/loans.json

```diff
@@ -3,7 +3,8 @@
     "barcode": "reroils1",
     "loans": {
       "active": 2,
-      "overdue": 1,
+      "overdue_active": 1,
```

Comment:
Do we use camel case in JSON files?

Reply:
Only in the SON of the resource `document`